src/main/java/com/auth0/client/mgmt/OrganizationsEntity.java

```diff
@@ -51,11 +50,9 @@ public Request<OrganizationsPage> list(PageFilter filter) {
         applyFilter(filter, builder);
 
         String url = builder.build().toString();
-        BaseRequest<OrganizationsPage> request = new BaseRequest<>(client, url, HttpMethod.GET, new TypeReference<OrganizationsPage>() {
-        });
 
-        request.addHeader(AUTHORIZATION_HEADER, "Bearer " + apiToken);
-        return request;
+        return new BaseRequest<>(client, tokenProvider, url,  HttpMethod.GET, new TypeReference<OrganizationsPage>() {
```

Comment:
Totally insignificant, of course, but presumably you didn't mean to double-space these parameters referencing `url` and/or `HttpMethod`?

Reply:
lol good catch, so many files and looks like I got the regex wrong in a few places 😆. I'll clean that up.

Reply:
new commit pushed that cleans this up